arm-logic/2016-06-01/swagger/logic.json

```diff
@@ -1151,1135 +1149,5009 @@
+      },
+      "description": "The sku name."
+    },
+    "WorkflowState": {
+      "type": "string",
+      "enum": [
+        "NotSpecified",
+        "Completed",
+        "Enabled",
+        "Disabled",
+        "Deleted",
+        "Suspended"
+      ],
+      "x-ms-enum": {
+        "name": "WorkflowState",
+        "modelAsString": false
+      }
+    },
+    "WorkflowStatus": {
+      "type": "string",
+      "enum": [
+        "NotSpecified",
+        "Paused",
+        "Running",
+        "Waiting",
+        "Succeeded",
+        "Skipped",
+        "Suspended",
+        "Cancelled",
+        "Failed",
+        "Faulted",
+        "TimedOut",
+        "Aborted",
+        "Ignored"
+      ],
+      "x-ms-enum": {
+        "name": "WorkflowStatus",
+        "modelAsString": false
+      }
+    },
+    "ParameterType": {
+      "type": "string",
+      "enum": [
+        "NotSpecified",
+        "String",
+        "SecureString",
+        "Int",
+        "Float",
+        "Bool",
+        "Array",
+        "Object",
+        "SecureObject"
+      ],
+      "x-ms-enum": {
+        "name": "ParameterType",
+        "modelAsString": false
+      }
+    },
+    "KeyType": {
+      "type": "string",
+      "enum": [
+        "NotSpecified",
+        "Primary",
+        "Secondary"
+      ],
+      "x-ms-enum": {
+        "name": "KeyType",
+        "modelAsString": false
+      }
+    },
+    "Sku": {
+      "type": "object",
+      "required": [
+        "name"
+      ],
+      "properties": {
+        "name": {
+          "$ref": "#/definitions/SkuName",
+          "description": "The name."
+        },
+        "plan": {
+          "$ref": "#/definitions/ResourceReference",
+          "description": "The reference to plan."
+        }
+      },
+      "description": "The sku type."
+    },
+    "ContentLink": {
+      "type": "object",
+      "properties": {
+        "uri": {
+          "type": "string",
+          "description": "The content link URI."
+        },
+        "contentVersion": {
+          "type": "string",
+          "description": "The content version."
+        },
+        "contentSize": {
+          "type": "integer",
+          "format": "int64",
+          "description": "The content size."
+        },
+        "contentHash": {
+          "$ref": "#/definitions/ContentHash",
+          "description": "The content hash."
+        },
+        "metadata": {
+          "$ref": "#/definitions/Object",
+          "description": "The metadata."
+        }
+      },
+      "description": "The content link."
+    },
+    "ContentHash": {
+      "type": "object",
+      "properties": {
+        "algorithm": {
+          "type": "string",
+          "description": "The algorithm."
+        },
+        "value": {
+          "type": "string",
+          "description": "The value."
+        }
+      },
+      "description": "The content hash."
+    },
+    "Correlation": {
+      "type": "object",
+     
```

Comment:
enum? How would the customer know what to put over here while creating the Map? How important is the usability of the API? REST API documentation will be generated from swagger. The description of the properties are not that helpful. Where do you expect the customer to  go to find out what can he/she provide over here?